build/build.yml

```diff
@@ -0,0 +1,116 @@
+pool:
+  name: Hosted macOS
+  demands:
+  - npm
+  - node.js
+
+steps:
+- task: ArchiveFiles@1
+  displayName: 'Archive source '
+  inputs:
+    rootFolder: '$(Build.SourcesDirectory)'
+    includeRootFolder: false
+    archiveType: tar
+    archiveFile: '$(Build.BinariesDirectory)/source.tar.gz'
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Publish Artifact: source'
+  inputs:
+    PathtoPublish: '$(Build.BinariesDirectory)/source.tar.gz'
+    ArtifactName: source
+
+- task: Npm@1
```

Comment:
We are using yarn now
We should replace all the npm commands with their equivalent yarn commands. Let me know if you need help with this.

Reply:
I'll check this in as is since that's what the build is currently doing - but I'm planning on spending some time cleaning this up during committer time.